web/src/lib/components/shared-components/context-menu/context-menu.svelte

```diff
@@ -65,9 +65,9 @@
     aria-label={ariaLabel}
     aria-labelledby={ariaLabelledBy}
     bind:this={menuElement}
-    class:max-h-[100vh]={isVisible}
-    class:max-h-0={!isVisible}
-    class="flex flex-col transition-all duration-[250ms] ease-in-out outline-none"
+    class="{isVisible
+      ? 'max-h-screen max-h-svh'
```

Comment:
I put both 100vh and 100svh for retro compatibility.
I preferred svh instead of dvh because it should be more stable reading the [CSS standard](https://www.w3.org/TR/css-values-4/#viewport-variants).

> The sizes of the [dynamic viewport-percentage units](https://www.w3.org/TR/css-values-4/#dynamic-viewport-percentage-units) are not stable even while the viewport itself is unchanged. Using these units can cause content to resize e.g. while the user scrolls the page. Depending on usage, this can be disturbing to the user and/or costly in terms of performance.